Measurement_2015.py

```diff
@@ -76,7 +76,7 @@ def DoDeviceSweep(GraphProc,rpg,DataFile,SweepInst,ReadInst,
 		TSet = -1,
 		Timeout = -1, Wait = 0.5,
 		ReturnData = False,MakePlot=True,
-		SocketDataNumber=2,					# 5 for 9T, 2 for Dilution fridge
+		SocketDataNumber=5,					# 5 for 9T, 2 for Dilution fridge
```

Comment:
16T still uses Measurement_2015 while DF uses Measurement_2015_v2. Remember to change SocketDataNumber when adapting to 16T later again.